src/App.js

```diff
@@ -1,10 +1,11 @@
 import "./App.css";
-import "./global.css";
-import SignUp from "./pages/SignUp/SignUp";
-import Waiting from "./pages/Waiting/Waiting";
+// import "./global.css";
```

Comment:
Please don't remove the code unless required
- please change the commented code to the original
- [ ] line 2 
- [ ] line 4 
- [ ] line 5